nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestColdStorage.java

```diff
@@ -109,6 +119,48 @@ public void shouldMoveToColdStorage() throws IOException {
         assertEquals("dummy", ((ManagedBlob) content).getProviderId());
     }
```

Comment:
I think we can group the tests within the permissions to:
- keep the name shorted
- group the tests related to same things on the same tests

as we have a helper method, i think we can use a name:

`shouldMoveToColdStorage` which can test with an Admin and after that with a regular user that has the right permission

Reply:
As we discussed, I'll wait for more input on this. I confess I'm not completely comfortable with this approach since we are violating some unit tests principles.
Obviously we can argue if we are doing unit or integration tests and it may influence the decision.

At the end I'll go with the more consensual approach but hoping that this discussion could help creating some awareness for this subjects 🙂